### ISSUE TITLE
Quickly subscribe to starred Repositories

### ISSUE THREAD

**Issue by lukaswelte** [librariesio/support#14]
This is related to #5.

It would be great to have a list of the starred (and maybe also watched) repositories available. As well as a option to watch all of them or just a little button beneath every list item to subscribe to one project with one click.

**Comment by andrew** [comment]
Yeah definitely, most likely we'll have the stars and watched projects separately with buttons to subscribe to all of them at once.
